scripts/west_commands/runners/nrfutil.py

```diff
@@ -121,8 +121,7 @@ def _append_batch(self, op, json_file):
         elif op_type == 'reset':
             cmd += ['--reset-kind', _op['kind']]
         elif op_type == 'erase':
-            cmd += ['--reset-kind', _op['kind']]
-
+                cmd.append(f'--{_op["kind"]}')
```

Comment:
Is GitHub's rendering failing or is this an indentation issue?

Reply:
Nope, you are right, thanks! Not sure how I managed to mangle this after testing it!